test/test_make_equidistant.py

```diff
@@ -47,7 +47,7 @@ def timestamps_values_uncertainties_kind(
         dt, kind for make_equidistant()
     """
     # Set the maximum absolute value for floats to be really unique in calculations.
-    float_abs_max = 1e128
+    float_abs_max = 1e300
```

Comment:
We should actually try to remove this completely. The previous hassle resulted from a check we performed on our own, which we removed already.

Reply:
Removing does not work, because it produces serious overflows.

Reply:
We should actually leave this around `1e64` to avoid those overflows in the calculation of the uncertainties.